backend.go

```diff
@@ -214,6 +214,12 @@ func (p *Backend) handleStartup() error {
 			return fmt.Errorf("error sending deny SSL request: %w", err)
 		}
 		return p.handleStartup()
+	case *pgproto3.GSSEncRequest:
```

Comment:
You can add another value to the previous case, if they're handled exactly the same way.